communication/youtube/youtube-guidelines.md

```diff
@@ -212,4 +215,5 @@ detailed information about streaming, see our [Streaming Config]
 [Community Meeting]: /events/community-meeting.md
 [Office Hours]: /events/office-hours.md
 [Meet our Contributors]: /mentoring/meet-our-contributors.md
-[Streaming Config]: ./youtube/streaming-config.md
\ No newline at end of file
+[Streaming Config]: ./youtube/streaming-config.md
+[Subprojects]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
```

Comment:
```suggestion
[Subprojects]: ./governance.md#subprojects
```

Reply:
Hi @nikhita! Thanks for the review, let me fix.